src/documentation/pages/components/footer/examples/_internal-brightspeed.vue

```diff
@@ -0,0 +1,182 @@
+<template lang="pug">
+  <ComponentExample title="Internal" id="internal-brightspeed" :tabs="exampleTabs">
+    p.-text(slot="example-description")
+      | Show the internal footer for authenticated users.
+    footer.chi-footer(slot='example')
+      .chi-footer__content
+        .chi-footer__internal
+          .chi-footer__internal-content.-mw--1200
+            .chi-dropdown.chi-footer__language
+              a.chi-button.-icon.-flat.-light.-sm.chi-dropdown__trigger(id="language-dropdown-button" ref="language-dropdown-button" data-position="top-start" aria-label="Select your preferred language")
+                .chi-button__content
+                  i.chi-icon.icon-globe-network(aria-hidden="true")
+                  span English
+              .chi-dropdown__menu.-w--sm.-text--body
+                a.chi-dropdown__menu-item.-active(href="#")
```

Comment:
Let's define a
`const FOOTER_LANGUAGE_DROPDOWN_ITEMS = [ ITEMS HERE ];`
and use it like:
`a(v-for="(item, index) in FOOTER_LANGUAGE_DROPDOWN_ITEMS" :key="index").chi-dropdown__menu-item.-active(href="#")= item`.

The array must be reused across all the examples.
Adding an element into the array should add the item in all the examples.
This way we achieve much better reusability and scalability.
@soujanreddy @josecarlosgonzalezv @jllr We need to keep this in mind for all the components and implementations of the new documentation system.